	Fix compatability with compaq compilers by ifdef'ing out some
	2.3.2 additions - <taviso@gentoo.org> (14 Jun 2003).
--- glibc-2.3.2/include/features.h
+++ glibc-2.3.2/include/features.h
@@ -285,7 +285,8 @@
 #if defined __GNUC__ \
     || (defined __PGI && defined __i386__ ) \
     || (defined __INTEL_COMPILER && (defined __i386__ || defined __ia64__)) \
-    || (defined __STDC_VERSION__ && __STDC_VERSION__ >= 199901L)
+    || (defined __STDC_VERSION__ && __STDC_VERSION__ >= 199901L) \
+    && !(defined(__DECC) || defined(__DECCXX))
 # define __GLIBC_HAVE_LONG_LONG	1
 #endif
 
--- glibc-2.3.2/linuxthreads/sysdeps/pthread/bits/pthreadtypes.h
+++ glibc-2.3.2/linuxthreads/sysdeps/pthread/bits/pthreadtypes.h
@@ -64,9 +64,11 @@
 {
   struct _pthread_fastlock __c_lock; /* Protect against concurrent access */
   _pthread_descr __c_waiting;        /* Threads waiting on this condition */
+#if !(defined(__DECC) || defined(__DECCXX)) /* hide this from DEC CC/CXX */
   char __padding[48 - sizeof (struct _pthread_fastlock)
 		 - sizeof (_pthread_descr) - sizeof (__pthread_cond_align_t)];
   __pthread_cond_align_t __align;
+#endif /* __DECC */
 } pthread_cond_t;
 
 
